package-lock.json

```diff
@@ -0,0 +1,1184 @@
+{
+  "name": "good-influx",
```

Comment:
that is due to npm v5. It's a new thing that will be included with each package.